sources/auth.c

```diff
@@ -646,6 +646,78 @@ static inline int od_auth_frontend_cert(od_client_t *client)
 	return -1;
 }
 
+static inline int od_auth_frontend_mdb_iamproxy(od_client_t *client)
```

Comment:
why do we make separate handler, not just another case in `od_auth_frontend_cleartext` ?